etl/data_helpers/__init__.py

```diff
@@ -0,0 +1,4 @@
+from etl.data_helpers.geo_old import calculate_region_sums
```

Comment:
If this function is not used in any data step, I'd suggest to simply remove it. Especially because we should not use a function that, by default, creates its own aggregate for the world. If a dataset contains data for the world (which is common) but we want to create aggregates for our own region definitions, this function can't be used.